client/src/app/+videos/+video-watch/comment/video-comment-add.component.scss

```diff
@@ -99,13 +99,21 @@ form {
   }
 }
 
-.table-emoji {
-  td {
-    &:first-child {
-      width: 20px;
+.emoji-flex {
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+
+  .emoji-flex-item {
+    text-align: left;
+    margin: auto;
+    width: 227px;
```

Comment:
here @kimsible can i suggest using % instead of fixing it at 227px? wide screens, narrow screens throw a fit and this

Reply:
Thanks @test2a for your comment. Fixing it at 227px allow to fit the longest markup (stuck_out_tongue...) What percent value would you use ?

Reply:
how about this https://jsfiddle.net/kdr9b1vg/
i am not at all an expert on anything, :worried:
i checked this on my phone using codepen and it worked somewhat.

Reply:
thanks @test2a for the snippet, I understand what you mean now.
In my case we need to update the width flex-item if any new markup is longer than current longest one.
The `margin: auto` on flex-item replaces the flex percent width I think. 
Also I'm not an expert too btw. :) I'll check see what I can do.